81-IDevID_and_IAK_support.md

```diff
@@ -218,13 +228,14 @@ A potential workflow is presented below for the scenario using EK, EK certificat
 9. The agent gets its UUID in case of a successful registering process.
 
 ### Registration from the registrar service perspective when interacting with the agent
-1. The main endpoints here are do_register_agent() and do_activate_agent() invoked by the agent.
+<!-- TODO: add the specific API changes here./ -->
+1. The main endpoints here are do_register_agent() and do_activate_agent() invoked by the agent. The registration will affect the /v{api_version}/agents/{agent_id} exchange, while the activation will affect the /v{api_version}/agents/{agent_id}/activate endpoint. The specific changes might not happen at the API endpoints, but on the data exchanged and the api_version to support the new information.
 2. Decode data and parse certificates. This implies modifying the method doRegisterAgent to support new parameters regarding IAK and IDevID. This method exercises a POST to /v{api_version}/agents/{agent_id} at registrar_common.py do_POST method.  
 3. Verify IDevID certificate chain of trust + IAK certificate chain of trust
 4. Create a challenge to send to the agent for prove the possession of the private keys. It's important to mention that the challenge will change from using Make/ActivateCredential to use TPM2_Certify.
 4. Create a challenge to send to the agent for prove the possession of the private keys. 
 5. Receive the challenge response and process
-6. Generate the certificates for LAK and LDevID keys with the owner/company's CAs. 
+6. Load and check the certificates for LAK and LDevID keys with the owner/company's CAs. The certificates should be generated on a separate process.
```

Comment:
"The certificates should be generated on a separate process" -> "The certificates should be generated using a separate process"

Reply:
done.